docs/05-error-reporting.md

```diff
@@ -29,7 +29,7 @@ Only proper releases should be handled correctly by sentry. This means we do no
 
 ## Upload of source maps
 
-Sentry wants source maps in order to display stack traces properly. We are not uploading them automatically. Instead you can [use the cli](https://docs.sentry.io/platforms/javascript/sourcemaps/#uploading-source-maps-to-sentry) to upload source maps.
+Sentry wants source maps in order to display stack traces properly. We are not uploading them automatically. Instead, you can [use the cli](https://docs.sentry.io/platforms/javascript/sourcemaps/#uploading-source-maps-to-sentry) to upload source maps.
```

Comment:
```suggestion
Sentry needs source maps in order to display stack traces properly. We are not uploading them automatically. Instead, you can [use the cli](https://docs.sentry.io/platforms/javascript/sourcemaps/#uploading-source-maps-to-sentry) to upload source maps.
```
Is this even necessary anymore? Isn't it uploaded automatically with each realease?